quattor/functions/filesystem.pan

```diff
@@ -49,6 +49,32 @@ function lvs_add = {
     SELF;
 };
 
+@documentation{
+    desc = Adds a list of logical volumes to a volume group with mutiple lv options. To replace function lvs_add which only sets size.
```

Comment:
Sorry if this is obvious but is the implication should this deprecate lvs_add and people encouraged to move over? (I see your comment and agree this is for discussion)

Reply:
Not sure how the two releases of template standard and core are released, whether they are released together? If they are, then this PR will solve any compatibility issue, even if lvs_add is modified. 
https://github.com/quattor/template-library-standard/pull/151

so no need to modify line 987 in above pr if lvs_add is modified to take list of dict as 2nd arg instead of long value

Reply:
i would add the comment of relplacing lvs_add as a comment in the pan code, not part of the docs. as you mention, it's a one time thing to do

Reply:
ok, make sense. done.